main.py

```
@@ -47,6 +48,36 @@ def create_wallet():
     return sk, pk
 
 
+def request_chain(network, start_index, limit):
+    """Broadcast a chain_request for blocks starting at start_index."""
+    req = {"type": "chain_request", "data": {"start_index": start_index, "limit": limit}}
+    asyncio.create_task(network._broadcast_raw(req))
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)